docs/content/intro/tutorial03.md

```diff
@@ -65,11 +65,3 @@ The next time you push to the repository, the webhook system should trigger a bu
 > For more on configuring GitHub, see [the GitHub Guide](../topics/github.md)
 
 After configuring Brigade to test new features, read [part 4 of this tutorial][part4] to write a new feature to the uuid-generator project, which will trigger a test build using Brigade.
-
----
-
-Prev: [Part 2][part2] `|` Next: [Part 4][part4]
-
-[part2]: tutorial02.md
-[part4]: tutorial04.md
-[brigade-github-app]: https://github.com/Azure/brigade-github-app
```

Comment:
Some or all of lines 73-75 need to stay, as long as they are referenced in-line above (line 67 for example)